core/src/main/java/org/eclipse/hono/config/ProtocolAdapterProperties.java

```diff
@@ -59,7 +59,8 @@ public final boolean isAuthenticationRequired() {
      * <p>
      * The default value of this property is {@code true}.
      *
-     * @param authenticationRequired {@code true} if the server should wait for downstream connections to be established during startup.
+     * @param authenticationRequired {@code true} if the protocol adapter demands the authentication of devices to allow
```

Comment:
how about just
> `true` if the protocol adapter should require clients to authenticate

The description already puts it into the right context, FMPOV

Reply:
I have changed it now.